keyboards/inett_studio/sqx/universal/rules.mk

```diff
@@ -2,29 +2,22 @@
 MCU = atmega32u4
 
 # Bootloader selection
-#   Teensy       halfkay
-#   Pro Micro    caterina
-#   Atmel DFU    atmel-dfu
-#   LUFA DFU     lufa-dfu
-#   QMK DFU      qmk-dfu
-#   ATmega32A    bootloadHID
-#   ATmega328P   USBasp
 BOOTLOADER = atmel-dfu
 
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = lite 	# Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
 MOUSEKEY_ENABLE = no        # Mouse keys
-EXTRAKEY_ENABLE = yes 		# Audio control and System control
+EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 # Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
 SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
-NKRO_ENABLE = yes 			# USB Nkey Rollover
-UNICODE_ENABLE = no         # Unicode
-RGBLIGHT_ENABLE = yes        # Use RGB underglow light
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
 RGB_MATRIX_ENABLE = yes     # Use RGB per key light
 
-LTO_ENABLE = yes			# Reducing firmware size
+LTO_ENABLE = yes            # Reducing firmware size
```

Comment:
```suggestion
LTO_ENABLE = yes            # Reducing firmware size

LAYOUTS = 60_ansi 60_hhkb 60_iso 60_ansi_split_bs_rshift 60_tsangan_hhkb
```

Reply:
added those layouts will cause the compilation failed:
```
Making inett_studio/sqx/universal with keymap unxmaal                                                  [ERRORS]
 |
 | quantum/keymap_common.c:169:46: error: type of 'fn_actions' does not match original declaration [-Werror=lto-type-mismatch]
 |  __attribute__((weak)) const uint16_t PROGMEM fn_actions[] = {
 |                                               ^
 | layouts/community/60_iso/unxmaal/keymap.c:110:24: note: array types have different bounds
 |  const uint16_t PROGMEM fn_actions[] = {
 |                         ^
 | layouts/community/60_iso/unxmaal/keymap.c:110:24: note: 'fn_actions' was previously declared here
 | lto1.exe: all warnings being treated as errors
 | lto-wrapper.exe: fatal error: C:\msys64\home\leiyu\qmk_utils\avr8-gnu-toolchain\bin\avr-gcc.exe returned 1 exit status
 | compilation terminated.
 | c:/msys64/home/leiyu/qmk_utils/avr8-gnu-toolchain/bin/../lib/gcc/avr/8.3.0/../../../../avr/bin/ld.exe: error: lto-wrapper failed
 | collect2.exe: error: ld returned 1 exit status
 |
make[1]: *** [tmk_core/rules.mk:306：.build/inett_studio_sqx_universal_unxmaal.elf] 错误 1
Making inett_studio/sqx/universal with keymap via                                                      [OK]
Making inett_studio/sqx/universal with keymap xyverz                                                   [OK]
Making inett_studio/sqx/universal with keymap yanfali                                                  [ERRORS]

 * The firmware is too large! 29604/28672 (932 bytes over)

make[1]: *** [tmk_core/rules.mk:411：check-size] 错误 1
Making inett_studio/sqx/universal with keymap yanfali_wkl                                              [ERRORS]
cc1.exe: error: users/yanfali_wkl: not a directory [-Werror]
cc1.exe: all warnings being treated as errors
 |
 |
 |
make[1]: *** [tmk_core/rules.mk:386：.build/obj_inett_studio_sqx_universal_yanfali_wkl/keyboards/inett_studio/sqx/universal/universal.o] 错误 1
Make finished with errors
```